rules/pkg-normalize.js

```diff
@@ -22,6 +22,10 @@ const isGitRepository = repository => {
 		return false;
 	}
 
+	if (typeof repository !== 'string' && !Array.isArray(repository) && Object.prototype.hasOwnProperty.call(repository, 'url')) {
```

Comment:
Is that possible? If not, maybe we should first check if repository is a `string` or an `object`. If not, throw an error or something?